cypressAuto/cypress/integration/UI_stepsdefinitions/Home.js

```diff
@@ -0,0 +1,24 @@
+import { Given, When, Then } from 'cypress-cucumber-preprocessor/steps'
+import Home from '../UI_PageObject/HomePage'
+import fixtures from '../../fixtures/constantValues.json'
+const home = new Home

```

Comment:
Why it is required? can't you use imported variable name

Reply:
but for class i need to create object